src/operation-node/operation-node-transformer.ts

```diff
@@ -825,4 +831,9 @@ export class OperationNodeTransformer {
     // An Object.freezed leaf node. No need to clone.
     return node
   }
+
+  protected transformExplain(node: ExplainNode): ExplainNode {
+    // An Object.freezed leaf node. No need to clone.
+    return node
```

Comment:
`ExplainNode` isn't a leaf node as it has node children. You need to transform them.

Reply:
Nice catch! done!